docs/devguide/dev_docs.rst

```diff
@@ -96,7 +96,7 @@ DataFlowKernel
 ==============
 
 .. autoclass:: parsl.dataflow.dflow.DataFlowKernel
-   :members:  __init__, submit, cleanup, load_checkpoints, config, handle_update, launch_task, _count_deps, _count_all_deps, sanitize_and_wrap, _load_checkpoints
+   :members:  __init__, submit, cleanup, load_checkpoints, checkpoint, config, handle_update, launch_task, _count_deps, _count_all_deps, sanitize_and_wrap, _load_checkpoints
```

Comment:
Isn't this just all the members? Unless we're trying to purposely exclude members, I think we should consider
```
:members:
```
instead of
```
:members:  __init__, submit, cleanup, load_checkpoints, checkpoint, config, handle_update, launch_task, _count_deps, _count_all_deps, sanitize_and_wrap, _load_checkpoints
```
because it is easier to maintain so that we don't have to remember to add a member here if we add one to the code. That applies everywhere...